types/api.d.ts

```diff
@@ -32,6 +32,7 @@ export interface DocumentSubmissionResponse
     | 'userUpdatedBy'
   > {
   createdAt: string;
+  validUntil: DateTime;
```

Comment:
I think this should be optional, like my comment on https://github.com/LBHackney-IT/evidence-api/pull/98#discussion_r660753129

Reply:
Similar to the evidence-api PR, I don't think `validUntil` needs to be optional